src/app/cli/src/cli_commands.rs

```diff
@@ -206,10 +206,7 @@ pub fn get_command(
             } else {
                 odf_server::AccessTokenStoreScope::Workspace
             },
-            // TODO: improve URL parser
-            submatches
-                .get_one::<String>("server")
-                .map(|s| Url::parse(s).unwrap()),
+            parse_server_url(submatches.get_one::<String>("server"))?,
```

Comment:
Can I also ask for a small command behavior change?

Can we please make the `server` argument positional instead of a named, so we could write:
```
kamu login platform.kamu.dev
```
instead of:
```
kamu login --server platform.kamu.dev
```
I really want to keep this command similar to `docker login`.

Reply:
Done